sdk/storage/storage-blob/samples/typescript/basic.ts

```diff
@@ -18,28 +12,21 @@ async function main() {
   // Use SharedKeyCredential with storage account and account key
   const sharedKeyCredential = new SharedKeyCredential(account, accountKey);
 
-  // Use RawTokenCredential with OAuth token.  You can find more
-  // TokenCredential implementations in the @azure/identity library
-  // to use client secrets, certificates, or managed identities for
-  // authentication.
-  const tokenCredential = new RawTokenCredential("token");
-  tokenCredential.token = "renewedToken"; // Renew the token by updating token field of token credential
+  // Use TokenCredential with OAuth token
+  // const tokenCredential = new RawTokenCredential("token");
```

Comment:
Please also keep the comments about raw token credential / identity that added by David recently.

Reply:
Duplicate it in all the files ? 
or 
Only in this `basic.ts`?

Reply:
sigh the bad thing about duplication.  For now just not removing it.  Let's wait for Xiaoning's comments.

Reply:
Agree. At least we should recommend customers to use identity to get a TokenCredential, instead of using RawTokenCredential

---
In reply to: [297343197](https://github.com/Azure/azure-sdk-for-js/pull/4058#discussion_r297343197) [](ancestors = 297343197)

Reply:
If am correct, @azure/identity works on for Node.js? Does it suit with browsers?

If @azure/identity totaly meets the OAuth credential scenario, we should always recommend customers using identity package.

---
In reply to: [297345781](https://github.com/Azure/azure-sdk-for-js/pull/4058#discussion_r297345781) [](ancestors = 297345781)